CHANGELOG.md

```diff
@@ -1,5 +1,417 @@
 # Changelog
 
+## v0.71.0-rc.0
```

Comment:
just a reminder that we need to also add the RC1 commits https://github.com/facebook/react-native/releases/tag/v0.71.0-rc.1

Reply:
And for RC2: https://github.com/facebook/react-native/releases/tag/v0.71.0-rc.2

Reply:
And for RC3: https://github.com/facebook/react-native/releases/tag/v0.71.0-rc.3